conda/environments/cuml_dev_cuda10.2.yml

```diff
@@ -36,6 +37,7 @@ dependencies:
 - nbsphinx
 - recommonmark
 - faiss-proc=*=cuda
+- rapids-build-env=0.15*
```

Comment:
We probably want this in all the versions of the dev environments, but we might want to do this in a follow up pr. This meta package contains other packages that are listed explicitly in this yaml if I’m not mistaken, which could lead to conflicts if they get out of sync, so a follow up PR that removes those packages might be the best way forward

Reply:
This was totally accidental. I made this change to the yaml file while I was testing the cmake changes and didn't remove it.